README.md

```diff
@@ -75,6 +75,7 @@ to something other than a raw file it should be marked with an asterisk.*
 | [`language_make`](plugins/language_make.lua?raw=1)                                                             | Syntax for the Make build system language                                                                                                                                                                                                                                                       |
 | [`language_meson`](plugins/language_meson.lua?raw=1)                                                           | Syntax for the [Meson](https://mesonbuild.com) build system language                                                                                                                                                                                                                            |
 | [`language_moon`](plugins/language_moon.lua?raw=1)                                                             | Syntax for the [MoonScript](https://moonscript.org) scripting language                                                                                                                                                                                                                          |
+| [`language_nelua`](https://github.com/AKDev21/nelua-lite-xl)                                                   | Syntax for [Nelua](http://nelua.io/) programming                                                                                                                                                                                                                                        |
```

Comment:
As this links to a repo and not directly to a file, would you mind adding an asterisk after the link?
```suggestion
| [`language_nelua`](https://github.com/AKDev21/nelua-lite-xl)\*                                                 | Syntax for [Nelua](http://nelua.io/) programming                                                                                                                                                                                                                                        |
```

Reply:
sure